platform/linux-generic/odp_packet_io.c

```diff
@@ -529,6 +536,27 @@ int odp_pktio_close(odp_pktio_t hdl)
 	return 0;
 }
 
+static int configure_tx_event_compl(pktio_entry_t *entry)
+{
+	odp_pool_info_t info;
+	odp_pool_param_t params;
+
+	if (odp_pool_info(entry->s.pool, &info))
+		return -1;
+
+	odp_pool_param_init(&params);
+
+	params.type = ODP_POOL_PACKET;
+	params.pkt.num = info.pkt.max_num;
+	params.pkt.len = 1;
+	entry->s.tx_compl_pool = odp_pool_create("_odp_pktio_tx_compl_pool", &params);
```

Comment:
Pool name could include pktio index. It would help debugging when pools have unique names.